agent/client.go

```diff
@@ -142,10 +164,10 @@ func (c *CanaryClient) send(method string, uri string, body []byte) ([]byte, err
 	// if the request fails for whatever reason, keep
 	// trying to reach the server
 	err = backoff.Retry(func() error {
-		log.Debug("Request: %s %s", method, uri)
```

Comment:
I upgraded my local copy of `github.com/op/go-logging`. I think that's what happened.